### PR TITLE
[JBWS-4038] added log.trace stmts to catch blocks marked with comment…

### PR DESCRIPTION
**Pull request by rsearls**
…, ignore

### CODE REVIEW

File: modules/server/src/main/java/org/jboss/wsf/stack/cxf/RequestHandlerImpl.java [hunk 1]
```diff
@@ -64,6 +65,7 @@
  */
 public class RequestHandlerImpl implements RequestHandler
 {
+   private static Logger log = Logger.getLogger(RequestHandlerImpl.class);
```
Comment (asoldano) [inline modules/server/src/main/java/org/jboss/wsf/stack/cxf/RequestHandlerImpl.java]: final

File: modules/server/src/main/java/org/jboss/wsf/stack/cxf/deployment/WSDLFilePublisher.java [hunk 2]
```diff
@@ -61,6 +62,7 @@
  */
 public class WSDLFilePublisher extends AbstractWSDLFilePublisher
 {
+   private static Logger log = Logger.getLogger(WSDLFilePublisher.class);
```
Comment (asoldano) [inline modules/server/src/main/java/org/jboss/wsf/stack/cxf/deployment/WSDLFilePublisher.java]: final

File: modules/server/src/main/java/org/jboss/wsf/stack/cxf/transport/JBossWSDestinationRegistryImpl.java [hunk 3]
```diff
@@ -36,7 +37,7 @@
  */
 public class JBossWSDestinationRegistryImpl extends DestinationRegistryImpl
 {
-
+   private static Logger log = Logger.getLogger(JBossWSDestinationRegistryImpl.class);
```
Comment (asoldano) [inline modules/server/src/main/java/org/jboss/wsf/stack/cxf/transport/JBossWSDestinationRegistryImpl.java]: final

**Comment by rsearls** [comment]
Requested changes complete.

**Pull request merged**